.github/workflows/memory-testing.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: Memory Testing
+
+on:
+  pull_request:
+    paths:
+      - "crates/bitwarden-crypto/**"
+      - "crates/memory-testing/**"
+  push:
+    paths:
+      - "crates/bitwarden-crypto/**"
+      - "crates/memory-testing/**"
+    branches:
+      - "main"
+      - "rc"
+      - "hotfix-rc"
+
+defaults:
+  run:
+    shell: bash
```

Comment:
❓ Is this necessary?

Reply:
We're running a shell script so I just set it by default, but the script isn't doing anything that specifically requires bash, I've removed it and it seems to work.